src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -69,12 +72,12 @@ function cell_diffusion_timescale(closure::AnisotropicBiharmonicDiffusivity, dif
     max_κy = maximum_numeric_diffusivity(closure.κy)
     max_κz = maximum_numeric_diffusivity(closure.κz)
 
-    return min(Δx^2 / max_νx,
-               Δy^2 / max_νy,
-               Δz^2 / max_νz,
-               Δx^2 / max_κx,
-               Δy^2 / max_κy,
-               Δz^2 / max_κz)
+    return min(Δx^4 / max_νx,
```

Comment:
Was the power of 2 from before a mistake?

Reply:
It certainly did seem like a mistake. What do you think?

Reply:
Yes, given that we have 4-th order derivatives,  think powers of 4 make more sense to me.  I do wonder whether there should be cross terms as we have something like \partial_xx \nabla^2, which makes me think that maybe a \Delta x^2 \Delta y^2 might be important, but if the spatial resolutions are similar I don't suppoose it would make much of a difference.

Reply:
That's an interesting point. The main issue that's raised by your question is that we aren't actually deriving this time scale. It might be better to leave this out of the code...

Reply:
I think what you are changing to is better than what we have currently, and I suspect that these numbers are going to be good enough so don't have a problem moving forward with this.